src/core/server/logging/layouts/json_layout.ts

```diff
@@ -46,20 +46,24 @@ export class JsonLayout implements Layout {
 
     return {
       message: error.message,
-      name: error.name,
-      stack: error.stack,
+      type: error.name,
+      stack_trace: error.stack,
     };
   }
 
   public format(record: LogRecord): string {
     return JSON.stringify({
       '@timestamp': moment(record.timestamp).format('YYYY-MM-DDTHH:mm:ss.SSSZ'),
-      context: record.context,
-      error: JsonLayout.errorToSerializableObject(record.error),
-      level: record.level.id.toUpperCase(),
       message: record.message,
-      meta: record.meta,
-      pid: record.pid,
+      error: JsonLayout.errorToSerializableObject(record.error),
+      log: {
+        level: record.level.id.toUpperCase(),
+        logger: record.context,
```

Comment:
How ES handles pattern layout for a given `meta` object? Does it enforces ECS compatible property name usage in `conversion patterns`? https://logging.apache.org/log4j/log4j-2.2/manual/layouts.html#PatternLayout

Reply:
For now the field usage is best effort and under migration.  They'll eventually move to prefixing custom fields with es.

https://github.com/elastic/elasticsearch/blob/master/test/logger-usage/src/main/java/org/elasticsearch/test/loggerusage/ESLoggerUsageChecker.java does some static checking, but there's no strict compliance yet

Reply:
> For now the field usage is best effort and under migration. They'll eventually move to prefixing custom fields with es.

What is our plan for this?